### PR TITLE
Update Font Awesome

### PR DESCRIPTION
**Pull request by janno42**
This updates font awesome to version 6, replaces a few icons (no more sad faces), and repairs weird column widths on the student index page.

### CODE REVIEW

**Review by He3lixxx — APPROVED** [review]
I tried to come up with a review comment that makes excessive use of icons and emojis, but failed within a reasonable time frame. Probably a job for @niklasmohrin.

File: evap/staff/templates/staff_questionnaire_index_list.html [hunk 1]
```diff
@@ -41,16 +41,16 @@
...
+                                    <button data-visibility="0" type="button" onclick="changeVisibility($(this));" class="btn btn-sm btn-light{% if questionnaire.visibility == 0 %} active{% endif %}" data-bs-toggle="tooltip" data-bs-placement="top" title="{% blocktrans %}Hide in lists{% endblocktrans %}"><span class="fas fa-fw fa-eye-slash" aria-hidden="true"></span></button>
+                                    <button data-visibility="1" type="button" onclick="changeVisibility($(this));" class="btn btn-sm btn-light{% if questionnaire.visibility == 1 %} active{% endif %}" data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans 'Show only for managers' %}"><span class="fas fa-fw fa-briefcase" aria-hidden="true"></span></button>
+                                    <button data-visibility="2" type="button" onclick="changeVisibility($(this));" class="btn btn-sm btn-light{% if questionnaire.visibility == 2 %} active{% endif %}" data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans 'Show for managers and editors' %}"><span class="fas fa-fw fa-person-chalkboard" aria-hidden="true"></span></button>
```
Comment (niklasmohrin) [inline evap/staff/templates/staff_questionnaire_index_list.html:46]: Gotcha! Sneakily swapping icons here

**Review by niklasmohrin — APPROVED** [review]

File: evap/staff/templates/staff_questionnaire_index_list.html (hunk 1 above)
Comment (niklasmohrin) [reply evap/staff/templates/staff_questionnaire_index_list.html:46]: (I like it though)

Comment (janno42) [reply evap/staff/templates/staff_questionnaire_index_list.html:46]: as the commit message states, it is about changing some icons :)

**Pull request merged**